lib/tests/union-bridge-methods.js

```diff
@@ -0,0 +1,695 @@
+      // Assert transfer permissions remains unchanged
+      await assertUnionTransferredPermissions(REQUEST_PERMISSION_DISABLED, RELEASE_PERMISSION_DISABLED);
+      await assertNoEventWasEmitted(txReceipt);
+    });
+
+    it("should setUnionTransferPermissions second vote be successful when voting to enable both permissions", async () => {
+      const txReceipt = await setUnionTransferPermissions(REQUEST_PERMISSION_ENABLED, RELEASE_PERMISSION_ENABLED, changeTransferPermissionsAuthorizer2Address, assertSuccessfulResponseCode);
+      // Assert
+      await assertUnionTransferredPermissions(REQUEST_PERMISSION_ENABLED, RELEASE_PERMISSION_ENABLED);
+      await assertLogUnionTransferPermissionsSet(txReceipt, changeTransferPermissionsAuthorizer2Address, REQUEST_PERMISSION_ENABLED, RELEASE_PERMISSION_ENABLED);
+    });
+
+    it("should requestUnionBridgeRbtc be successful after force pause", async () => {
+      const unionBridgeContractAddress = await getUnionBridgeContractAddress();
+      const weisTransferredBeforeRequest = await getWeisTransferredToUnionBridgeBalance();
+      const unionBridgeBalanceBeforeRequest = await getUnionBridgeBalance();
+      // Act
+      const txReceipt = await requestUnionBridgeRbtc(AMOUNT_TO_REQUEST, assertSuccessfulResponseCode);
+      // Assert
+      const expectedWeisTransferred = new BN(weisTransferredBeforeRequest).add(new BN(AMOUNT_TO_REQUEST));
+      const expectedUnionBridgeBalance = new BN(unionBridgeBalanceBeforeRequest).add(new BN(AMOUNT_TO_REQUEST));
+      await assertUnionBridgeBalance(expectedUnionBridgeBalance);
+      await assertWeisTransferredToUnionBridgeBalance(expectedWeisTransferred);
+      await assertLogUnionRbtcRequested(txReceipt, unionBridgeContractAddress, AMOUNT_TO_REQUEST);
+    });
+
+    it("should releaseUnionBridgeRbtc be successful after force pause", async () => {
+      // Arrange
+      const unionBridgeContractAddress = await getUnionBridgeContractAddress();
+      const weisTransferredBeforeRelease = await getWeisTransferredToUnionBridgeBalance();
+      const unionBridgeBalanceBeforeRelease = await getUnionBridgeBalance();
+      // Act
+      const txReceipt = await releaseUnionBridgeRbtc(AMOUNT_TO_RELEASE, assertSuccessfulResponseCode);
+      // Assert
+      const expectedWeisTransferredAfter = new BN(weisTransferredBeforeRelease).sub(new BN(AMOUNT_TO_RELEASE));
+      const expectedUnionBridgeBalanceAfter = new BN(unionBridgeBalanceBeforeRelease).sub(new BN(AMOUNT_TO_RELEASE));
+      await assertWeisTransferredToUnionBridgeBalance(expectedWeisTransferredAfter);
+      await assertUnionBridgeBalance(expectedUnionBridgeBalanceAfter);
+      await assertLogUnionRbtcReleased(txReceipt, unionBridgeContractAddress, AMOUNT_TO_RELEASE);
+    });
+  });
+}
+
+const importAccounts = async (privateKeys) => {
```

Comment:
I know there's a `importAccounts` function in `change-federation.js`. I think it's time for this function to be put into an util module and reuse it.